lib/wx/api_gen/gen_util.erl

```diff
@@ -225,17 +225,17 @@ erl_gl_copyright() ->
     erl_copyright_template("Apache-2.0 AND SGI-B-2.0").
 
 erl_copyright() ->
-    erl_copyright("Apache-2.0").
+    erl_copyright_template("Apache-2.0").
 
 erl_wx_copyright() ->
-    erl_copyright("Apache-2.0 AND LicenseRef-scancode-wxwindows-free-doc-3").
+    erl_copyright_template("Apache-2.0 AND LicenseRef-scancode-wxwindows-free-doc-3").
 
 erl_copyright_template(License) ->
     StartYear = start_year(get(current_class)),
     {CurrentYear,_,_}   = erlang:date(),
     w("%%~n",[]),
-    w("%% %CopyrightBegin%
-    w("%%
+    w("%% %CopyrightBegin%~n",[]),
+    w("%%~n",[]),
```

Comment:
I am not sure how this has worked before without closing the `"`, so I decided to close them.